Formula/seexpr.rb

```diff
@@ -24,45 +24,45 @@ class Seexpr < Formula
 
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DUSE_PYTHON=FALSE"
+      system "cmake", "..", *std_cmake_args, "-DCMAKE_INSTALL_RPATH=#{rpath}", "-DUSE_PYTHON=FALSE"
       system "make", "doc"
       system "make", "install"
     end
   end
 
   test do
-    actual_output = shell_output("#{bin}/asciigraph2").lines.map(&:rstrip).join("\n")
+    actual_output = shell_output("#{bin}/asciigraph2 'x^3-8*x'").lines.map(&:rstrip).join("\n")
```

Comment:
The previous (default) expression was `$val=.5*PI*x; 7*sin(val)/val`, but it causes some platforms (e.g. ARM) to fail test due to `x=0` being undefined and producing different output.

For simplicity, just use the polynomial from upstream README which should hopefully be identical across platforms.

Reply:
We might be able to fix this on Linux now too.  It was just missing some dependencies.  I'm looking into that now.

Reply:
Test failed on Linux due to case-sensitivity
```suggestion
    actual_output = shell_output("#{bin}/asciiGraph2 'x^3-8*x'").lines.map(&:rstrip).join("\n")
```

Reply:
Whoops, don't know how I missed that.